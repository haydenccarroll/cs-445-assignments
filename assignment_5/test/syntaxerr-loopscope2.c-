// C-F22
main()
{
     int x;

     while true do {
           int x;          // OK
           }

     for forever {
           int x;         // OK
           }

     for y = 1 to 10 do {
           int x;         // OK
           }

     for y = 1 to 10 do { {
           int x;        // OK
           } }

     for y = 1 to 10 do {
           int y;        // y redeclared
           }

     for y = 1 to 10 do { {
           int y;        // OK
           } }

     for y = 1 to 10 do ; // should NOT effect the next compound
     { int x; }          // OK
}

