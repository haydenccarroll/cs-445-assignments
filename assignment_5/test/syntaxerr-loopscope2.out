====================================
FILE: syntaxerr-loopscope2.c-
ERROR(10): Syntax error, unexpected '{', expecting '='.
Number of warnings: 0
Number of errors: 1
